query.go

```diff
@@ -56,7 +56,7 @@ func (fb *Firebase) EndAt(value string) *Firebase {
 func (fb *Firebase) OrderBy(value string) *Firebase {
 	c := fb.copy()
 	if value != "" {
-		c.params.Set(orderByParam, escapeString(value))
+		c.params.Set(orderByParam, fmt.Sprintf(`%q`, strings.Trim(value, `"`)))
```

Comment:
I was wondering about this, but as orderBy must take a string parameter, I thought there's not much point in going back and forth with types. (Note that I haven't changed the parameter type either.)

But sure, we can call that too. It would still work :)